docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/dependencies.md

```diff
@@ -0,0 +1,70 @@
+| strapi-helper-plugin               | ✨ @strapi/helper-plugin               |
+| strapi-plugin-users-permissions    | ✨ @strapi/plugin-users-permissions    |
+| strapi-plugin-i18n                 | ✨ @strapi/plugin-i18n                 |
+| strapi-plugin-upload               | ✨ @strapi/plugin-upload               |
+| strapi-plugin-documentation        | ✨ @strapi/plugin-documentation        |
+| strapi-plugin-graphql              | ✨ @strapi/plugin-graphql              |
+| strapi-plugin-email                | ✨ @strapi/plugin-email                |
+| strapi-plugin-sentry               | ✨ @strapi/plugin-sentry               |
+| strapi-plugin-content-type-builder | ✨ @strapi/plugin-content-type-builder |
+| strapi-plugin-content-manager      | ✨ @strapi/plugin-content-manager      |
+| strapi-provider-upload-rackspace   | ✨ @strapi/provider-upload-rackspace   |
+| strapi-provider-email-nodemailer   | ✨ @strapi/provider-email-nodemailer   |
+| strapi-provider-upload-local       | ✨ @strapi/provider-upload-local       |
+| strapi-provider-email-mailgun      | ✨ @strapi/provider-email-mailgun      |
+| strapi-provider-upload-cloudinary  | ✨ @strapi/provider-upload-cloudinary  |
+| strapi-provider-email-sendmail     | ✨ @strapi/provider-email-sendmail     |
+| strapi-provider-email-amazon-ses   | ✨ @strapi/provider-email-amazon-ses   |
+| strapi-provider-upload-aws-s3      | ✨ @strapi/provider-upload-aws-s3      |
+| strapi-provider-email-sendgrid     | ✨ @strapi/provider-email-sendgrid     |
+| create-strapi-starter              | create-strapi-starter                 |
+| create-strapi-app                  | create-strapi-app                     |
+| strapi-generate-policy             | strapi-generate-policy                |
+| strapi-generate-controller         | strapi-generate-model                 |
+| strapi-generate-model              | strapi-generate-controller            |
+| strapi-generate-new                | strapi-generate-new                   |
+| strapi-generate                    | strapi-generate                       |
+| strapi-generate-api                | strapi-generate-api                   |
+| strapi-generate-plugin             | strapi-generate-plugin                |
+| strapi-generate-service            | strapi-generate-service               |
+| strapi-connector-mongoose          | ❌ (removed from v4)                   |
+| strapi-connector-bookshelf         | ❌ (removed from v4)                   |
+| strapi-hook-redis                  | ❌ (removed from v4)                   |
+| strapi-hook-ejs                    | ❌ (removed from v4)                   |
+| strapi-middleware-views            | ❌ (removed from v4)                   |
+
+Example:
+
+If the `package.json` file of a Strapi v3 application has the `“strapi-plugin-upload”: “3.6.8”` dependency declaration, and the migration targets Strapi v4.0.7, the dependency declaration should be replaced with `“@strapi/plugin-upload”: “4.0.7”`.
```

Comment:
See above note, this is one package that isn't required to be manually set anymore.

I'd use `@strapi/strapi` here as an example